### PR TITLE
Project modal

### PR DESCRIPTION
**Pull request by awaisanwar544**
Added pop-up window (Modal) for mobile and desktop versions.

**Desktop version**
Screenshot 2022-01-26 at 4 11 33 PM

**Mobile version**
Screenshot 2022-01-26 at 4 12 42 PM

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -71,6 +71,36 @@ <h1 class="center-text primary-text-color primary-font heading u-margin">Hey the
...
+                </div>
+                
+            </div>
+        </div>
+
+        <!--- modal end --->
+
         <div id="items-container">
             <div class="work-item">
                 <div class="rounded-img">
```
Comment (andersonlebon) [inline app/index.html]: - [ ] Nice job on using Javascript to populate this HTML file with information coming from your js file however, The requirements say that all the code concerning `my work section` should be created using Javascript and be rendered when the page loads. This 👇  is what the requirements state
 
> The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.** Check examples of students' projects:  [here](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and [here 👇 ](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96.)

File: app/index.html [hunk 2]
```diff
@@ -71,6 +71,36 @@ <h1 class="center-text primary-text-color primary-font heading u-margin">Hey the
     <section id="works" class="u-margin">
         <h2 class="primary-text-color primary-font heading">My Recent works</h2>
         <div class="indicator"></div>
+
+        <!--- modal start --->
+        <div id="modal-wrapper">
+            <div id="modal" class="primary-font primary-text-color">
+                <div id="modal-snapshot">
+                    <div id="modal-cancel">
+                        <img src="/app/assets/icn-cancel-white.png" alt="cancel button">
```
Comment (andersonlebon) [inline app/index.html]: - [ ] Kindly try to fix this image cancel icon link, actually, the icon is not showing up. check here :point what I can see

image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @awaisanwar544 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 3]
```diff
@@ -18,10 +18,16 @@
...
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.25.4",
+    "hint": "^6.1.3",
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^21.0.0",
     "stylelint-csstree-validator": "^1.9.0",
     "stylelint-scss": "^3.21.0"
+  },
+  "dependencies": {
+    "eslint-config-airbnb": "^19.0.4"
```
Comment (MrBrN197) [inline package.json:31]: - [x] [OPTIONAL] please kindly remove this line on your package.json. You already have `eslint-config-airbnb` as a `devDependency` so this is extraneous.

**Review by MrBrN197 — APPROVED** [review]
## Hi, @awaisanwar544  👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MrBrN197 — COMMENTED** [review]
## Hi, @awaisanwar544  👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**